.github/workflows/build.yml

```diff
@@ -25,6 +25,9 @@ jobs:
                   go-version: ${{ matrix.go }}
                   cache: false
 
+            - name: Make prep
```

Comment:
I don't see `prep` added as a rule in the makefile within this diff.

Reply:
Prep already existed and there was no change to it. I.e. it's exists but not in the diff